Editors/TiledEditor/source/TiledLevel.hx

```diff
@@ -91,21 +96,68 @@ class TiledLevel extends TiledMap
 		}
 	}
 	
-	public function loadObjects(state:PlayState)
+	public function loadObjects()
```

Comment:
There isn't really a "right" or "wrong"; it just seems unnecessary in this case.